src/service/viewer-impl.js

```diff
@@ -71,6 +71,15 @@ const TRUSTED_VIEWER_HOSTS = [
 ];
 
 /**
+ * These domains are trusted for CORS impression requests
+ *
+ * @export {!Array<!RegExp>}
+ */
+const TRUSTED_REFERRER_HOSTS = [
+  /^t.co$/,
```

Comment:
Please add description to `t.co`, please refer to what we did with `TRUSTED_VIEWER_HOSTS`

Reply:
updated